jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantWidget/ExtendedVariantWidget.js

```diff
@@ -138,7 +126,10 @@ export default jbrowse => {
         return displayJSX
     }
 
-    function makeChart(samples, ref, alt, classes){
+    function makeChart(samples, feat, classes){
```

Comment:
This is a good change, but is outside the scope of general code cleanup. In the future changes like this should be in their own commits for the purposes of tracking development progress and for reverting changes if needed later down the line. Ok for now though. #63

Reply:
This is not exactly a huge change and it's a lot more important that the software work robustly. The PR can be re-titled 'code cleanup and bugfixing'